pages/common/omz.md

```diff
@@ -0,0 +1,32 @@
+# omz
+
+> Oh My Zsh command-line tool.
+> More information: <https://github.com/ohmyzsh/ohmyzsh>.
+
+- Update Oh My Zsh:
+
+`omz update`
+
+- Print the changes from the latest update of Oh My Zsh:
+
+`omz changelog`
+
+- Restart the current Zsh session and Oh My Zsh:
+
+`omz reload`
+
+- List plugins:
+
+`omz plugin list`
+
+- Enable/Disable an Oh My Zsh plugin:
+
+`omz plugin {{enable|disable}} {{plugin}}`
+
+- List themes:
```

Comment:
```suggestion
- List all available themes:
```

Reply:
@CleanMachine1 , same here, could you send the suggestions?
```suggestion
- List all themes:
```